com.ibm.streamsx.topology/opt/python/packages/streamsx/rest_primitives.py

```diff
@@ -1166,10 +1242,91 @@ def get_view_obj(_view, rc):
                     return view
     return None
 
-
 class StreamingAnalyticsService(object):
     """Streaming Analytics service running on IBM Bluemix cloud platform.
     """
+    def __init__(self, rest_client, credentials):
+        # If IAM, create a V2 delegator, if basic http auth, create a V1 delegator
+        #
+        # Delegators are required because we need to keep StreamingAnalyticsService
+        # around for backwards compatibility, yet it also needs to work with basic
+        # and IAM authentication.
+        if 'v2_rest_url' in credentials:
+            self._delegator = _StreamingAnalyticsServiceV2Delegator(rest_client, credentials)
+        else:
+            self._delegator = _StreamingAnalyticsServiceV1Delegator(rest_client, credentials)
+
+    def cancel_job(self, job_id=None, job_name=None):
+        self._delegator.cancel_job(job_id=job_id, job_name = job_name)
+
+    def start_instance(self):
```

Comment:
The docstring comments for the methods have been "lost", not copied from the v1 delegator they were "moved" to.